RELEASING.md

```diff
@@ -0,0 +1,106 @@
+# Releasing
+
+When an update to SPF needs to be released, follow these steps.
+You will need admin privileges for the GitHub [youtube/spfjs][]
+repo.
+
+
+## Prepare the new version
+
+1.  Ensure all code has been tested. Verify changes from the
+    previous version manually and make sure all unit tests pass.
+
+2.  Update the `version` property in the `package.json` and
+    `bower.json` files to increment the number using
+    [semantic versioning][].
```

Comment:
This wording is a little odd. How about: 

Increment the `version` property in the `package.json` and `bower.json` files using semantic versioning.